docs/execution-providers/Vitis-AI-ExecutionProvider.md

```diff
@@ -1,8 +1,7 @@
 ---
 title: AMD - Vitis AI
 description: Instructions to execute ONNX Runtime on AMD devices with the Vitis AI execution provider
-grand_parent: Execution Providers
-parent: Community-maintained
+parent: Execution Providers
 nav_order: 6
 redirect_from: /docs/reference/execution-providers/Vitis-AI-ExecutionProvider
```

Comment:
Update the redirection

Reply:
This is how all other EPs at this level are redirtected.  Not sure I understand the ask here

Reply:
Oh sure - when we move a page, you should add a redirect from the previous location (so that the old location can still be accessed via direct links and search). All of the EP docs were previously moved from the location shown above. This was at least a year ago, so any page equity from the previous location has probably diminished (I haven't looked at whether you can have multiple redirect_from fields). I would say for now, just add the location from which you moved this doc.